requirements.txt

```diff
@@ -1,3 +1,3 @@
-PyLint==1.6.5
-pep8>=1.6.0,<=1.6.2
-modernize==0.5
+PyLint>=1.7.1
+pycodestyle>=2.3.1
+modernize>=0.5
```

Comment:
Fair point regarding lower bounds. Since the dependencies (other than modernize, which hasn't seen an update in well over a year) utilize semver perhaps setting ranges around minor revisions makes sense here rather than hard pinning versions? Happy to have any changes to deps coordinated through the proper channels.